pivotal-api-test/target/test-classes/features/stories/EditStory.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Update a story in project from pivotal tracker
+
```

Comment:
This feature file and RemoveStory.feature have the same precondition (Background)
Please join them.